operator/src/main/helm/coherence-operator/dashboards/cache-details-dashboard.json

```diff
@@ -777,7 +777,7 @@
           "decimals": 2,
           "link": true,
           "linkTooltip": "Drill down to member details for  ${__cell}",
```

Comment:
There are 2 spaces between for and ${__cell}.  Should one be removed?  
I notice this on all text with ${__cell?} in it.